.github/workflows/checks.yaml

```diff
@@ -85,18 +79,14 @@ jobs:
           sudo apt-get update -y
           sudo apt-get install -y bats
 
-      - name: Use image built from main
-        run: |
           # Prepare quick_start.yaml
           make update-istio-quickstart-version
 
           # Make docker image available to k8s
           kind load docker-image openpolicyagent/opa:latest-istio
-        if: matrix.variant != 'tag'
 
       - name: Run e2e tests
         run: make test-e2e
-        timeout-minutes: 10
```

Comment:
Is there some default here already that we don't need this anymore ?

Reply:
Hmm why did I do that. 🤔 I'll revert and merge.

Reply:
Ah I remember. I haven't removed intentionally, it got removed reverting the other commit. I'll keep it instead! Good catch.